docs/api-reference/catalogd-webserver.md

```diff
@@ -13,6 +18,21 @@ As an example, to access the full FBC via the v1 API endpoint (indicated by path
 
 the URL to access the service would be `https://catalogd-service.olmv1-system.svc/catalogs/operatorhubio/api/v1/all`
 
+2. `api/v1/metas` endpoint that allows clients to retrive filtered portions of the FBC. 
+
+The metas endpoint accepts additional parameters following the pattern `/api/v1/metas?<parameters>`, and accepts parameters that corresponds to any combination of the fields of the [FBC Meta structure](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#schema) (except for `Blob`).
```

Comment:
Worth extending the example to chained parameters?  
```suggestion
The metas endpoint accepts additional parameters following the pattern `/api/v1/metas?<parameter>[&parameter...]`, and accepts parameters that corresponds to any combination of the fields of the [FBC Meta structure](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#schema) (except for `Blob`).
```

Reply:
Done.